WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerActivity.java

```diff
@@ -256,8 +256,6 @@ public void onRefreshStarted() {
                             mSwipeToRefreshHelper.setRefreshing(false);
                             return;
                         }
-                        mDispatcher.dispatch(SiteActionBuilder.newRemoveWpcomAndJetpackSitesAction());
```

Comment:
`onSiteRemoved` should still be called when a self-hosted site is removed. I'll double-check and get back to you on that.